app/src/main/java/de/danoeh/antennapod/view/LockableBottomSheetBehavior.java

```diff
@@ -47,28 +47,28 @@ public boolean onTouchEvent(CoordinatorLayout parent, V child, MotionEvent event
 
     @Override
     public boolean onStartNestedScroll(CoordinatorLayout coordinatorLayout, V child, View directTargetChild,
-                                       View target, int nestedScrollAxes) {
+                                       View target, int axes, int type) {
         boolean handled = false;
 
         if (!isLocked) {
-            handled = super.onStartNestedScroll(coordinatorLayout, child, directTargetChild, target, nestedScrollAxes);
+            handled = super.onStartNestedScroll(coordinatorLayout, child, directTargetChild, target, axes, type);
```

Comment:
I think this class should also filter calls to deprecated methods. Does the old method internally call the new method?

Reply:
>Does the old method internally call the new method?

Nope.

Reply:
What if a library function on the outside still calls the old method? It is then no longer blocked properly. Have you tested the change?

Reply:
That isn't possible as far as I know. The old methods are only called if the type is ViewCompat.TYPE_TOUCH, which isn't used anywhere.

As a side note, the old methods are in AndroidX CoordinatorLayout, while the new ones are in the Google Material library (in BottomSheetBehavior).

I've fixed this deprecation before in Slide; I tested and everything worked as it was supposed to (though YMMV).